_includes/layout/after-site-header.html

```diff
@@ -3,3 +3,9 @@
   <a href="http://devdocs.magento.com/guides/v2.0/release-notes/ReleaseNotes2.0.18CE.html" target="_blank">Magento 2.0.18</a> is the final 2.0.x release. After March 2018, Magento 2.0.x will no longer receive security patches, quality fixes, or documentation updates.</br> To maintain your site's performance, security, and PCI compliance, upgrade to the latest version of Magento.</a>
 </div>
 {% endif %}
+
+{% if page.group == "page-builder" %}
+<div class="message-banner">
+  Please note: The <strong>BlueFoot content to Page Builder data migration tool<strong> is not yet released. Stay tuned for more information.
```

Comment:
The phrasing is a little awkward. Consider rewriting:

> Documentation about migrating data from Bluefoot to Page Builder is not yet available. Stay tuned for more information.

Reply:
Yeah, it's more than just the documentation that isn't available, it's the module/tool itself. How about:
> The module for migrating BlueFoot content to Page Builder is not yet available. Stay tuned for more information.

Reply:
Sounds good!